code/game/objects/objs.dm

```diff
@@ -63,6 +66,85 @@
 		visible_message("<span class='danger'>[src] shatters into a million pieces!</span>")
 		qdel(src)
 
+/obj/proc/start_pulling(atom/movable/AM,gs)

```

Comment:
@nicbn is completely right here, it should be moved to `/atom/movable`

Just saying it doesn't work isn't an excuse. You're a coder, make it work.